test/lib/test-ringpop-cluster.js

```diff
@@ -118,10 +120,14 @@ function testRingpopCluster(opts, name, test) {
 
     tape(name, function onTest(assert) {
         var cluster = bootstrapClusterOf(opts, function onBootstrap(results) {
-            test(results, cluster, assert);
+            var oldEnd = assert.end.bind(assert);
+
+            assert.end = function newEnd() {
```

Comment:
You can also listen on the end event on assert.

This might be cleaner then monkey patching assert.end